tests/LocalDateRangeTest.php

```diff
@@ -219,6 +219,42 @@ public function testToString()
         ));
     }
 
+    /**
+     * @dataProvider providerToDatePeriod
+     *
+     * @param string $dateTime The date-time string that will be parse()d by LocalDate.
+     * @param string $expected The expected output from the native DateTime object.
+     */
+    public function testToDatePeriod($a, $expectedStart, $expectedEnd)
```

Comment:
Please add `string` types, and rename `$a` to `$dateRange`.

Reply:
Done :)